configuration/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@azure-iot/configuration",
+  "version": "0.0.1",
+  "description": "Provides configuration support for Azure IoT microservices",
+  "main": "configuration.ts",
+  "typings": "configuration.d.ts",
+  "author": "",
+  "dependencies": {
+    "mongodb": "^2.1.18",
+    "underscore": "^1.8.3"
+  },
+  "typings": "configuration.d.ts",
+  "scripts": {
+    "lint": "tslint **/*ts",
+    "build": "tsc",
+    "prepublish": "tsd install && npm run build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
I would set lint task as a prebuild otherwise you can publish your code without linting.